lv_core/lv_group.c

```diff
@@ -334,10 +340,25 @@ void lv_group_set_refocus_policy(lv_group_t * group, lv_group_refocus_policy_t p
 }
 
 static void lv_group_refocus(lv_group_t *g) {
-	if(g->refocus_policy == LV_GROUP_REFOCUS_POLICY_NEXT)
-		lv_group_focus_next(g);
-	else if(g->refocus_policy == LV_GROUP_REFOCUS_POLICY_PREV)
-		lv_group_focus_prev(g);
+    /*Refocus must temporarily allow wrapping to work correctly*/
+    uint8_t temp_wrap = g->wrap;
+
```

Comment:
I think a line is missing from here: `g->wrap = 1;` to enable wrap temporarily.

Reply:
Good catch, dunno how I missed that!